cfme/tests/configure/test_start_page.py

```diff
@@ -0,0 +1,104 @@
+import pytest
+import fauxfactory
+
+from cfme.base.credential import Credential
+from cfme.configure.settings import Visual
+from cfme.configure.access_control import Role, Group, User
+from cfme.utils.appliance.implementations.ui import navigate_to
+
+
+faux_value = fauxfactory.gen_alphanumeric()
+pytestmark = pytest.mark.usefixtures('browser')
+
+
+def get_landing_page():
+    view = navigate_to(Visual, 'All')
+    landing_pages = view.visualstartpage.show_at_login.all_options
+    start_page = []
+    for landing_page in landing_pages:
+        start_page.append(landing_page.text)
+    return start_page
+
+
+@pytest.yield_fixture(scope="module")
+def rbac_user(appliance):
+    role = Role(name='test{}'.format(faux_value))
+    role.create()
+    role_name = 'test' + faux_value
+    group = Group(description='test{}'.format(faux_value), role=role_name)
+    group.create()
+    group_user = Group('test{}'.format(faux_value))
+    username = 'test{}'.format(faux_value.lower())
+    password = 'test'
+    cred = Credential(principal=username, secret=password)
+    user = User(name='test{}'.format(faux_value),
+                credential=cred, group=group_user)
+    user.create()
+    appliance.server.logout()
+    user_login(user, appliance)
+    view = navigate_to(Visual, 'All')
+    values = view.visualstartpage.show_at_login.all_options
+    yield user, values
+    appliance.server.logout()
+    login_page = navigate_to(appliance.server, 'LoginScreen')
+    login_page.login_admin()
+    user.delete()
+    group.delete()
+    role.delete()
+
+
+def user_login(user, appliance):
+    login_page = navigate_to(appliance.server, 'LoginScreen')
+    login_page.log_in(user)
+
+
+def set_landing_page_rbac(user, values, appliance):
```

Comment:
**Required:**  This and set_landing_page_admin can be one function, they are almost the same